evap/evaluation/models_logging.py

```diff
@@ -128,6 +128,19 @@ class LoggedModel(models.Model):
     class Meta:
         abstract = True
 
+    def save(self, *args, **kw):
```

Comment:
Why was the order changed here? Does ruff expect "public" methods before "private" methods, or what is the criterion?

Reply:
Yes, it's [DJ102](https://docs.astral.sh/ruff/rules/django-unordered-body-content-in-model/). Also suggested by the [django style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#model-style)